.gitmodules

```diff
@@ -181,6 +181,10 @@
 	path = Freecad-Built-in-themes-beta
 	url = https://github.com/MisterMakerNL/Freecad-Built-in-themes-beta
 	branch = main
+[submodule "FreeCAD Stream Deck Addon"]
```

Comment:
The Addon Manager expects the submodule name entry to match, the path, so this should be "freecad_streamdeck_addon"